### PR TITLE
Community Page Changes / Conformance Badging

### PR DESCRIPTION
**Pull request by robmoffat**
- Added Badges
- Electron FDC3 -> FDC3 Sail
- Added NexJ, AdaptableTools to the front page / community page
- Added FINOS App Directory to top menu bar
- Added links to conformance project on front page

### CODE REVIEW

File: website/pages/en/community.js [hunk 1]
```diff
@@ -60,9 +62,27 @@ function Implementation({ type, title, publisher, image, infoLink, docsLink, bad
 					{docsLink ? <a href={docsLink} key={docsLink} className="button">Documentation</a> : null}
 				</div>
 				<div className="prose" dangerouslySetInnerHTML={{ __html: description }}></div>
+				<div className='conformance'>
```
Comment (mattjamieson) [inline website/pages/en/community.js:65]:
```suggestion
				<div className="conformance">
```

File: website/data/community.json [hunk 2]
```diff
@@ -118,6 +140,21 @@
...
+      "conformance" : [
+        {
+          "src": "/img/community/certified-1.2.png",
+          "items": [
+            {
+              "text": "Glue42 Enterprise v3.17.0 / Glue FDC3 v2.6.0 Certified Conformant With FDC3 1.2 Specification on 2nd December 2022",
+              "link": "https://www.finos.org/blog/first-fdc3-1.2-certified-desktop-agents"
+            },
+            {
+              "text": "Glue42 Core+ v1.0 Browser Extension v1.4 / Glue FDC3 v2.6.0 Certified Conformant With FDC3 1.2 Specification on 6th December 2022 ",
```
Comment (mattjamieson) [inline website/data/community.json:152]:
```suggestion
              "text": "Glue42 Core+ v1.0 Browser Extension v1.4 / Glue FDC3 v2.6.0 Certified Conformant With FDC3 1.2 Specification on 6th December 2022",
```

File: website/pages/en/community.js [hunk 3]
```diff
@@ -60,9 +62,27 @@ function Implementation({ type, title, publisher, image, infoLink, docsLink, bad
...
+					{
+						conformance ? <h3><a title="The FDC3 Conformance Framework" href="https://github.com/finos/FDC3-Conformance-Framework">Conformance Details</a></h3> : null
+					}
+					{
+						(conformance ? conformance :[]).map(c =>
+							<div className='conformance-element'>
```
Comment (mattjamieson) [inline website/pages/en/community.js:71]:
```suggestion
							<div className="conformance-element">
```

**Review by mattjamieson — APPROVED** [review]
LGTM, proposed a couple of super-minor changes.

File: docs/fdc3-charter.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ title: FDC3 Charter
 Financial desktop applications include any app used in common financial workflows:
 
 * Traditional native applications implemented in C++, .NET, Java, Python, etc.
-* Hybrid web/native applications - stand alone native apps embedding Chromium (e.g. Electron, CEF, NW.js)
+* Hybrid web/native applications - stand alone native apps embedding Chromium (e.g. Electron, CEF, NW.js, FDC3-Sail)
 * Desktop web applications - platform based apps extending Chromium (e.g. OpenFin, Finsemble, Glue42)
```
Comment (kriswest) [inline docs/fdc3-charter.md:11]: These two lines are already wrong and I think this edit is compounding it. We use the definition of hybrid web/native apps elsewhere and none of the listed examples actually fit it. Electron and FDC3-Sail should be on the next line (Desktop web applications), which itself needs a tweak, 

```suggestion
* Hybrid web/native applications - stand-alone native apps embedding Chromium (e.g. a .NET application embedding WebView using CEF or WebView2)
* Desktop web applications - Web applications running in a desktop container (e.g. OpenFin, Finsemble, Glue42, Electron, FDC3-Sail)
```

File: website/data/users.json [hunk 5]
```diff
@@ -151,5 +151,19 @@
         "infoLink": "https://www.flextrade.com/",
         "pinned": true,
         "isMember": true
+    },
+    {
```
Comment (kriswest) [inline website/data/users.json:155]: Not alpha sorted - seems other recent additions weren't either, needs fixing.

File: website/data/community.json [hunk 6]
```diff
@@ -17,11 +17,11 @@
       "description": "<p>ChartIQ is a powerful and flexible HTML5 charting library with millions of users worldwide. Written in JavaScript, it runs entirely within the browser. With state of the art integrations, including  third-party apps, educational components, analysis, and more, you can create efficient workflows and streamline the way to work.</p><p>ChartIQ works on any platform (mobile, web, desktop) and any framework (Angular, React).</p><p>Most companies have at least one financial charting library for each platform or app they target—web, C#, Java, Android, iOS, etc., so developers have multiple code bases to maintain. With ChartIQ, write your code once and use it everywhere.</p>"
     },
     {
-      "title": "Electron-FDC3",
+      "title": "FDC3 Sail",
```
Comment (kriswest) [inline website/data/community.json:20]: Note community page implementation alpha sorts enties before rendering so ordering doesn't matter here (but does in users.json)

File: website/data/users.json (hunk 5 above)
Comment (kriswest) [reply website/data/users.json:155]: ...either by alpha sorting the data in the file or before it is rendered here:
https://github.com/finos/FDC3/blob/c2470b2f49eff4cbecf6d436af684a21c67b5874/website/pages/en/users.js#L25-L34

and here:
https://github.com/finos/FDC3/blob/c2470b2f49eff4cbecf6d436af684a21c67b5874/website/pages/en/index.js#L165

File: website/pages/en/community.js [hunk 7]
```diff
@@ -103,7 +123,7 @@ function ImplementationsShowcase() {
 function Implementations(props) {
 	const { config: siteConfig } = props;
 	const { repoUrl } = siteConfig;
-	const editUrl = `https://www.finos.org/get-involved-fdc3`;
+	const editUrl = `/get-involved`;
```
Comment (kriswest) [inline website/pages/en/community.js:126]: This page doesn't actually contain details of how to add your implementation to the page.

The original solution was this link (which will let you open a PR to add your details): https://github.com/finos/FDC3/edit/master/website/data/community.json

Either we'll need to update the get involved page, revert to the link, or drop in a mailto: link with a template asking for the relevant details.

**Review by kriswest — CHANGES_REQUESTED** [review]
Need to retain or fix alpha sorting of entries in user showcase (community page sorts for itself).  

Add your implementation link isn't quite right.

File: website/data/community.json (hunk 6 above)
Comment (robmoffat) [reply website/data/community.json:20]: added code to sort so we don't have to worry about doing this manually.

File: website/data/users.json (hunk 5 above)
Comment (robmoffat) [reply website/data/users.json:155]: done now I think

File: website/pages/en/community.js (hunk 7 above)
Comment (robmoffat) [reply website/pages/en/community.js:126]: I'll revert to the link

File: website/data/community.json (hunk 6 above)
Comment (kriswest) [reply website/data/community.json:20]: thank you!

File: website/data/community.json [hunk 8]
```diff
@@ -40,14 +44,7 @@
...
       "type": "platform-provider",
-      "badges": [
-        {
-          "text": "FDC3 1.2 Supported"
-        },
-        {
-          "text": "Open Source"
-        }
-      ],
+      "badges": [],
```
Comment (kriswest) [inline website/data/community.json:47]: Not keeping the open source tag?

**Review by kriswest — APPROVED** [review]
LGTM

File: website/data/community.json (hunk 8 above)
Comment (robmoffat) [reply website/data/community.json:47]: no - losing badges for desktop agents, it's just too cluttered

**Comment by robmoffat** [comment]
I'm still seeing "merging is blocked" due to your earlier review - can you try and push it along?

**Pull request merged**